GVRf/Framework/framework/src/main/java/org/gearvrf/GVRMesh.java

```diff
@@ -361,7 +361,14 @@ public GVRMesh getBoundingBox() {
     public List<GVRBone> getBones() {
         return mBones;
     }
-
+    /**
+     * Sets the status of mesh, whether static or dynamic, by default it is static.
```

Comment:
Please provide examples of when people should use dynamic, when static; advantages/disadvantages. We can't force them trying to figure what this does by reading the source code.